.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  env: {
```

Comment:
You need a ts parser like we do [here](https://github.com/gnosis/dex-react/blob/develop/.eslintrc.js#L2)
and maybe extend a plugin or two for TS

Reply:
Actually, not using the parser may be the reason for prettier problems

Reply:
Wrong PR 😓 
But add it for TS